cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1746,6 +1746,13 @@ def test_rerun_course_success(self):
         self.assertEqual(1, len(source_videos))
         self.assertEqual(source_videos, target_videos)
 
+    def test_rerun_course_resets_advertised_date(self):
```

Comment:
@adampalay why is that a requirement? Pylint explicitly ignores the docstring requirement on `test_` methods, the idea being that test names should be descriptive enough (diff-quality reports 100% here). I find it's pretty mixed throughout the code base on whether or not test methods have docstrings. In this file, for example, only some test cases have docstrings.

If it's a requirement we should change the Pylint to enforce it.